src/coreclr/vm/codeman.cpp

```diff
@@ -3215,21 +3215,22 @@ BYTE* EEJitManager::allocGCInfo(CodeHeader* pCodeHeader, DWORD blockSize, size_t
     } CONTRACTL_END;
 
     MethodDesc* pMD = pCodeHeader->GetMethodDesc();
```

Comment:
You can just pass in the `MethodDesc` instead of the whole `CodeHeader`. This method does not need the `CodeHeader` anymore.